### ISSUE TITLE
Error reproducing outputs for Manchester

### ISSUE THREAD

**Issue by Robinlovelace** [Hussein-Mahfouz/cycle-networks#13]
I get:

```
> graph_sf$cycle_infra[is.na(graph_sf$cycle_infra)] <- 0
Error in `[[<-.data.frame`(`*tmp*`, i, value = numeric(0)) : 
  replacement has 0 rows, data has 4176
> graph_sf$cycle_infra[is.na(graph_sf$cycle_infra)]
```

**Comment by Robinlovelace** [comment]
The culprit:

https://github.com/Hussein-Mahfouz/cycle-networks/blob/d9d655fc2cb7e6d4f377b63525c3565dc7563aa5/R/_5.0_identifying_cycle_infastructure_from_osm_tags.R#L235

Plan, only run that line if there are NA values.

**Comment by Robinlovelace** [comment]
Fix in commit above...

**Comment by Robinlovelace** [comment]
Hitting this now:

```
graph_sf_unweight <- infra_exists(graph=graph_sf_unweight, network=streetnet2)
Error in `[[<-.data.frame`(`*tmp*`, i, value = numeric(0)) : 
  replacement has 0 rows, data has 3102
Called from: `[[<-.data.frame`(`*tmp*`, i, value = numeric(0))
```

**Comment by Robinlovelace** [comment]
Error is that in `graph_sf` there is `cycle_infra.x` and `cycle_infra.y`...

**Comment by Robinlovelace** [comment]
All seems to be fixed now, in #14

**Comment by Robinlovelace** [comment]
This issue will auto-close when the PRs are merged.
